src/layout/layout_state.rs

```diff
@@ -10,6 +12,15 @@ pub struct LayoutState {
     pub components: Vec<ComponentState>,
     /// The direction which the components are laid out in.
     pub direction: LayoutDirection,
+    /// The font to use for the timer text. `None` means a default font should
+    /// be used.
+    pub timer_font: Option<Font>,
+    /// The font to use for the times and other values. `None` means a default
+    /// font should be used.
```

Comment:
Same thing here, not entirely sure:
```suggestion
    /// The font to use for the times and other numerical values. `None` means a
    /// default font should be used.
```